test/functional/wallet_listsinceblock.py

```diff
@@ -11,6 +11,10 @@
     assert_raises_rpc_error,
     connect_nodes,
 )
+from decimal import Decimal
+
+# Sequence number that is BIP 125 opt-in and BIP 68-compliant
+BIP125_SEQUENCE_NUMBER = 0xfffffffd
```

Comment:
That should already be in `test_framework.messages` module

Reply:
Right. Updated.